### PR TITLE
V14: refactor property editor aliases

### PR DESCRIPTION
**Pull request by Zeegaan**
# Notes
- Refactored richtext editors `TinyMCE` alias to `RichText` instead, to not leak technology

# How to test
- Be on v14/dev
- Create a document type with a rich text property (allow as root)
- Go to the template and insert the RichText `@Model.Value("{YourPropertyAliasHere}")`
- Create some content with that document type (type in some text in the rich text editor)
- Assert that you can see the frontend (localhost/44331) and the things you have typed in the rich text editor
- Swap to this branch
- It should still work 👍

### CODE REVIEW

**Review by elit0451 — CHANGES_REQUESTED** [review]
I am not sure if you have observed the same behaviour, but on a fresh install on `v14/dev`, when switching to your branch I get "Website is Under Maintenance" and I don't get the upgrade afterwards when I try to access the backoffice. Therefore, the Rich text editor is broken _(I cannot edit anything)_. Can you have a look?

This is what happens on my end and I get the error `No property editor 'Umbraco.TinyMCE' found, converting to a Label
System.NullReferenceException: The property editor with alias Umbraco.TinyMCE does not exist`.

https://github.com/umbraco/Umbraco-CMS/assets/21998037/6a1c2656-d926-4f50-aa74-9eb3e5946afa

File: src/Umbraco.Core/Constants-PropertyEditors.cs [hunk 1]
```diff
@@ -173,7 +173,7 @@ public static class Aliases
             /// <summary>
             ///     TinyMCE
             /// </summary>
-            public const string TinyMce = "Umbraco.TinyMCE";
+            public const string TinyMce = "Umbraco.RichText";
```
Comment (elit0451) [inline src/Umbraco.Core/Constants-PropertyEditors.cs:176]: Should we rename the name of the constant `TinyMce`, as well?

Comment (Zeegaan) [reply src/Umbraco.Core/Constants-PropertyEditors.cs:176]: Hmmm that would be breaking, maybe we should Create a new Constant called `RichText`, and then obsolete this one ? 🤔 

So something like:
```
public const string RichText = "Umbraco.RichText";

[Obsolete("please use RichText constant instead")]
public const string TinyMce = RichText;
```

Comment (elit0451) [reply src/Umbraco.Core/Constants-PropertyEditors.cs:176]: That would be ideal, yeah!

**Review by elit0451 — APPROVED** [review]
Looks good, merging shortly 💪

**Pull request merged**